Terminals.cs

```diff
@@ -1,525 +0,0 @@
-          /* VAR */
-          if (string.CompareOrdinal(lexeme, "VAR") == 0) {
-            return Token.VAR;
-          } /* end if */
-          break;
-
-      } /* end switch */
-      break;
-
-    case /* length = 4 */ 4 :
-      switch (lexeme[1]) {
-
-        case 'A' :
-          /* CASE */
-          if (string.CompareOrdinal(lexeme, "CASE") == 0) {
-            return Token.CASE;
-          } /* end if */
-          break;
-
-        case 'H' :
-          /* THEN */
-          if (string.CompareOrdinal(lexeme, "THEN") == 0) {
-            return Token.THEN;
-          } /* end if */
-          break;
-
-        case 'I' :
-          /* WITH */
-          if (string.CompareOrdinal(lexeme, "WITH") == 0) {
-            return Token.WITH;
-          } /* end if */
-          break;
-
-        case 'L' :
-          /* ELSE */
-          if (string.CompareOrdinal(lexeme, "ELSE") == 0) {
-            return Token.ELSE;
-          } /* end if */
-          break;
-
-        case 'O' :
-          /* LOOP */
-          if (string.CompareOrdinal(lexeme, "LOOP") == 0) {
-            return Token.LOOP;
-          } /* end if */
-          break;
-
-        case 'R' :
-          /* FROM */
-          if (string.CompareOrdinal(lexeme, "FROM") == 0) {
-            return Token.FROM;
-          } /* end if */
-          break;
-
-        case 'X' :
-          /* EXIT */
-          if (string.CompareOrdinal(lexeme, "EXIT") == 0) {
-            return Token.EXIT;
-          } /* end if */
-          break;
-
-        case 'Y' :
-          /* TYPE */
-          if (string.CompareOrdinal(lexeme, "TYPE") == 0) {
-            return Token.TYPE;
-          } /* end if */
-          break;
-
-      } /* end switch */
-      break;
-
-    case /* length = 5 */ 5 :
-      switch (lexeme[0]) {
-
-        case 'A' :
-          /* ARRAY */
-          if (string.CompareOrdinal(lexeme, "ARRAY") == 0) {
-            return Token.ARRAY;
-          } /* end if */
-          break;
-
-        case 'B' :
-          /* BEGIN */
-          if (string.CompareOrdinal(lexeme, "BEGIN") == 0) {
-            return Token.BEGIN;
-          } /* end if */
-          break;
-
-        case 'C' :
-          /* CONST */
-          if (string.CompareOrdinal(lexeme, "CONST") == 0) {
-            return Token.CONST;
-          } /* end if */
-          break;
-
-        case 'E' :
-          /* ELSIF */
-          if (string.CompareOrdinal(lexeme, "ELSIF") == 0) {
-            return Token.ELSIF;
-          } /* end if */
-          break;
-
-        case 'U' :
-          /* UNTIL */
-          if (string.CompareOrdinal(lexeme, "UNTIL") == 0) {
-            return Token.UNTIL;
-          } /* end if */
-          break;
-
-        case 'W' :
-          /* WHILE */
-          if (string.CompareOrdinal(lexeme, "WHILE") == 0) {
-            return Token.WHILE;
-          } /* end if */
-          break;
-
-      } /* end switch */
-      break;
-
-    case /* length 6 */ 6 :
```

Comment:
I had added reserved word OPAQUE here. You need to copy the section for all the reswords of length 6.